Mathlib/Geometry/Manifold/AnalyticManifold.lean

```diff
@@ -164,7 +165,7 @@ instance AnalyticManifold.prod {E A : Type} [NormedAddCommGroup E] [NormedSpace
 /-- Analytic manifolds are smooth manifolds. -/
 instance AnalyticManifold.smoothManifoldWithCorners [ChartedSpace H M]
```

Comment:
Please rename the instance now. I don't have a strong opinion if you need a deprecation; up to you.

Reply:
Instances are normally not deprecated.